test/puppetlabs/puppetdb/scf/storage_test.clj

```diff
@@ -967,10 +1000,12 @@
            {:title "/etc/foobar/baz"
             :tags #{"file" "class" "foobar"}
             :line 20}}
-         (-> (query-to-vec "SELECT title, tags, line FROM catalog_resources WHERE certname_id = (select id from certnames where certname = ?)" certname)
-             jdbc/convert-result-arrays
-             tags->set
-             set)))
+         (call-with-query-rows
+          [(str "select title, tags, line from catalog_resources"
+                 "  where certname_id = (select id from certnames"
+                 "                         where certname = ?)")
```

Comment:
even though it's test code, we should change the queries in this file to use joins instead of correlated subqueries